internal/report/report.go

```diff
@@ -1202,8 +1206,8 @@ func New(prof *profile.Profile, o *Options) *Report {
 		}
 		return measurement.ScaledLabel(v, o.SampleUnit, o.OutputUnit)
 	}
-	return &Report{prof, computeTotal(prof, o.SampleValue, o.SampleMeanDivisor),
-		o, format}
+	total, unexpNegSamples := computeTotal(prof, o.SampleValue, o.SampleMeanDivisor)
```

Comment:
unexpNegSamples -> hasNegativeSamples, here and in other places.

Reply:
Done. 
Not sure if this could be confusing, because -diff_base profiles will have negative samples, but this will be false.

Reply:
I see. Well, maybe leave the old name then. Up to you.

Reply:
Switching back to old name.